### ISSUE TITLE
Loss convergence损失收敛问题

### ISSUE THREAD

**Issue by 1129571** [MRzzm/DINet#74]
我该如何判断模型是否收敛？是否可以通过lr_g下降来判断？因为我使用的显卡显存只只有8G，必须要调小batch size，就不能按照readme中的参考epoch判断了。我在lr_g下降到大约原来一半的时候停止训练，得到的预测结果只有很少一部分可以对得上，并且训练集也不准确，貌似没有收敛。
How can I determine if the model is converging? Is it possible to use LR_ Judging by a decrease in g? Because the graphics card I am using only has 8GB of memory, it is necessary to reduce the batch size, so I cannot determine based on the reference epoch in the readme. I am in lr_ When g drops to about half of its original value, the training is stopped, and only a small portion of the predicted results can match, and the training set is also inaccurate, seemingly without convergence.
